kubernetes/structures_container.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func flattenCapability(in []v1.Capability) []string {
-	att := make([]string, 0, len(in))
+	att := make([]string, len(in), len(in))
```

Comment:
It is needed, because direct access via `[i]` requires elements to be already pre-allocated.

It wouldn't be necessary if we used `append()`, but that's unnecessary since we know precise length of the array. I would only use `append` when I don't know how many elements I'm going to have as it allocates and re-allocates the memory for you (i.e. possibly more than you really need).

https://play.golang.org/p/s9YliIhbQy
https://blog.golang.org/go-slices-usage-and-internals